includes/class-algolia-plugin.php

```diff
@@ -69,7 +69,7 @@ private function __construct() {
 
 		$this->compatibility = new Algolia_Compatibility();
 
-		add_action( 'init', array( $this, 'load' ), 20 );
+		add_action( 'init', array( $this, 'load' ), 5 );
```

Comment:
Interesting change. This would however break all implementations relying on the fact that this plugin was initially loaded at 5.

Reply:
Yep, it has some side-effects. However, a lower priority than standard would be preferable due to the large amount of plugins that just uses the default priority. 

I understand that you are not willing to take the risk, but it would be nice if the priority were changeable in some way. Proposing some solutions here: 

- A constant in PHP, predefined to 20 byt changeable if run before plugin execution. 
- A setting in the options panel (named compability mode or something)
- A filter that filters the priority of the above filter (might be a little weird). 

I have seen all of the above examples in other large plugins with compability issues like this, before. Do you want me to move forward on any of the above options or have a solution of how to achieve better compability with more plugins?